src/provider/bpmn/properties/NameProps.js

```diff
@@ -64,7 +64,8 @@ function Name(props) {
     },
     getValue: (element) => {
       return element.businessObject.name;
-    }
+    },
+    rows: 1
```

Comment:
I'd rather have an auto-resizing mechanism in place here (up to a certain height).